opslib/osm/pod.py

```diff
@@ -60,7 +60,7 @@ def ingress_resource(self):
             r["spec"]["tls"] = self.tls
         return r
 
-    def add_rule(self, hostname: str, service_name, port, path: str = "/"):
+    def add_rule(self, hostname: str, service_name, path_type, port, path: str = "/",):
```

Comment:
give it a default value of "Prefix". This way we don't need to change every charm.

Reply:
also, execute `black opslib/`.